### ISSUE TITLE
Bot response on already responded last message on startup

### ISSUE THREAD

**Issue by ArtemkaKun** [dariotarantini/vgram#12]
The problem can be reproduced with the example code too.

If I copy-paste example code to example.v file, then go to the console and run v run example.v - a bot will start successfully. After I print a command "/start" - I will get an expected response. 
But if then I will terminate the bot process via Ctrl + C and start it one more time - it will immediately respond on the last /start command one more time.

![image](https://user-images.githubusercontent.com/36485221/144477857-4f1a8e6b-365e-490d-80c7-20b678d0fd65.png)

**Comment by dariotarantini** [comment]
yep, thats happen because the library load also the last update. maybe the client should ignore it. i will update examples

**Comment by moixllik** [comment]
Increase +1 to last_offset in [hi_main.v](https://github.com/dariotarantini/vgram/blob/master/examples/hi_man.v#L11)
```v
updates = bot.get_updates(offset: last_offset + 1, limit: 100)
```
[API Telegram](https://core.telegram.org/bots/api#getupdates)
